sandbox-tools/merge-repos/README.md

```diff
@@ -15,22 +15,21 @@ This script performs the following
 - Clone the `opentelemetry/opentelemetry-sandbox-web-js` staging branch `auto-merge/repo-staging` (must exist) into a local temporary folder (removing any previous local folder first)
 - Removes any possible "conflicting" files from the root of the merge branch (if present -- only needed for local testing and first run)
 - Loops through each configured repository and
```

Comment:
This is updating the documentation to describe the changes that the script is now performing.